diff --git a/m4/gnulib-cache.m4 b/m4/gnulib-cache.m4
index acebc83f..7226ef86 100644
--- a/m4/gnulib-cache.m4
+++ b/m4/gnulib-cache.m4
@@ -44,6 +44,10 @@
 #  --avoid=getopt-posix-tests \
 #  --avoid=lock-tests \
 #  --avoid=tls-tests \
+#  --avoid=array-oset-tests \
+#  --avoid=avltree-oset-tests \
+#  --avoid=memchr-tests \
+#  --avoid=memset_explicit-tests \
 #  announce-gen \
 #  assert \
 #  attribute \
@@ -180,7 +184,7 @@ gl_MODULES([
   xvasprintf-posix
 ])
 gl_WITH_CXX_TESTS
-gl_AVOID([getopt-posix-tests lock-tests tls-tests])
+gl_AVOID([getopt-posix-tests lock-tests tls-tests array-oset-tests avltree-oset-tests memchr-tests memset_explicit-tests])
 gl_SOURCE_BASE([lib])
 gl_M4_BASE([m4])
 gl_PO_BASE([])
